packages/app/src/styles.ts

```diff
@@ -0,0 +1,3 @@
+// Entry point of `vite.styles.config.js` to build global styles for distribution
+import 'react-reflex/styles.css';
+import '@h5web/lib/dist/styles.css';
```

Comment:
Why not using `import ./styles.css` as in `lib` ?

Reply:
Because in this case, I want to import the distributed lib files, not the global lib styles. This is how we allow consumers to import only one CSS file when using `@h5web/app`. I have clarified the comments in `styles.ts/css` in both packages and replaced `import ./styles.css` with the full shared styles import in `lib/src/styles.ts` to avoid any confusion.

Reply:
Oh I see

Reply:
I missed this behaviour when documenting the package build process in `CONTRIBUTING.md`, sorry. I'm working on it.

Reply:
Done: https://github.com/silx-kit/h5web/pull/942/commits/b1eab4e873b87cf35835c75108848d364f46ddae